cmd/cost/list.go

```diff
@@ -142,6 +182,87 @@ func listCostsUnderOU(OU *organizations.OrganizationalUnit, awsClient awsprovide
 	return nil
 }
 
+type AccountCost struct {
+	AccountID string
+	Cost      float64
+	Unit      string
+}
+
+type OUCost struct {
+	Costs   []AccountCost
+	OU      *organizations.OrganizationalUnit
+	options *listOptions
+}
+
+func (o *OUCost) getCost(awsClient awsprovider.Client) error {
```

Comment:
Why not return the costs here are pass the array to the print function?

Reply:
We thought a type might be a better way than another function on the options struct and a function with 1 parameter better than a function with 3.

Reply:
The print function will only recieve The costs array
And the get needs to return an error maybe?